game/plugins/src/main/kotlin/gg/rsmod/plugins/content/items/pestle_and_mortar/pestleAndMortar.plugin.kts

```diff
@@ -11,7 +11,7 @@ sources.forEach { source ->
     on_item_on_item(Items.PESTLE_AND_MORTAR, source) {
         // If the player only has one of the source item, grind it immediately
         if (player.inventory.getItemCount(source) == 1) {
-            grindItem(player, source, 1)
+            grindItem(player, definitions[source]!!.result, 1)
```

Comment:
Is it necessary to use the not-null assertion operator (!!) in this case? 
Or is there a better way you could've written this?

https://kotlinlang.org/docs/null-safety.html#the--operator